server/auth-router.js

```diff
@@ -119,6 +119,11 @@ module.exports = function authRouter(config = {}) {
 			options.passwordless = true;
 		}
 
+		// Specify partnerContentfulPageId
+		if (req.query.partnerContentfulPageId) {
```

Comment:
Let's shorten this to `partnerContentId` please.

Reply:
done